wasm_src/ast_wrapper/post.ts

```diff
@@ -97,7 +97,7 @@ Module.setCanvas = function (canvas) {
 Module.plot = Module.cwrap("plotGrid", "number", ["number", "number", "number", "number", "number", "number", "number", "number", "number", "number", "number", "string"]);
 Module.emptyFitsChan = Module.cwrap("emptyFitsChan", "number");
 Module.putFits = Module.cwrap("putFits", null, ["number", "string"])
-Module.getFrameFromFitsChan = Module.cwrap("getFrameFromFitsChan", "number", ["number"]);
+Module.getFrameFromFitsChan = Module.cwrap("getFrameFromFitsChan", "number", ["number", "boolean"]);
```

Comment:
`cwrap` [only supports ](https://emscripten.org/docs/api_reference/preamble.js.html#cwrap)`number` and `string`. That's why we've used `number` elsewhere.